packages/ripple-nuxt-tide/modules/event/pages/search.vue

```diff
@@ -107,6 +107,14 @@ export default {
           values: setFilterDate.endOf('day').toISOString()
         }
       }
+      else {
+        const today = moment().endOf('day').toISOString()
```

Comment:
Please add timezone on the date, as a user browser may not in the same timezone as Victoria.